tests/e2e/docker/image_config.go

```diff
@@ -28,10 +28,10 @@ const (
 	// It should be uploaded to Docker Hub. OSMOSIS_E2E_SKIP_UPGRADE should be unset
 	// for this functionality to be used.
 	previousVersionOsmoRepository = "osmolabs/osmosis-dev"
-	previousVersionOsmoTag        = "v8.0.0-2-debug"
+	previousVersionOsmoTag        = "v9.0.0-debug"
 	// Pre-upgrade repo/tag for osmosis initialization (this should be one version below upgradeVersion)
-	previousVersionInitRepository = "osmolabs/osmosis-init"
-	previousVersionInitTag        = "v8.0.0-4-osmo"
+	previousVersionInitRepository = "osmolabs/osmosis-e2e-init-chain"
+	previousVersionInitTag        = "v9.0.0"
```

Comment:
just making sure - is the v9.x branch updated with the most recent "init" changes?

Reply:
I did, thanks for checking. We still need to work on automating this as well so we don't need to manually create this every time.

Reply:
L, I lied, it wasn't 😞 thanks for calling this out